packages/@aws-cdk/aws-codepipeline/test/pipeline.test.ts

```diff
@@ -485,6 +486,298 @@ describe('', () => {
       });
     });
   });
+
+  describe('cross account key alias name tests', () => {
+    const kmsAliasResource = 'AWS::KMS::Alias';
```

Comment:
I would just inline this variable, honestly 😛. It's not like this will change.

Reply:
Yeah, but writing it inline means that VS Code doesn't add the suggestion and then I have to type the whole thing over and over again. I'm far too lazy for that.